src/routes/chapter.ts

```diff
@@ -12,22 +12,22 @@ const controller = new ChaptersController();
 
 export function addRoutes() {
      app.get("/chapters", async (req: Request, res: Response) => {
-          await controller.getAllChapters(req, res);
+          return await controller.getAllChapters(req, res);
```

Comment:
why do we need a return statement ?

Reply:
I was trying to fix an old error, that's why I put the return keyword. After solving the problem (which was not linked to the "return" keyword), I decided to put return everywhere (because there were some functions with return and other without it).
But it does not change anything, the function will end there with or without it.

Reply:
Ok not a problem I guess we can leave them here